test/JSON-Schema-Test-Suite/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ if(JSON_SCHEMA_TEST_SUITE_PATH)
             JSON-Suite::Optional::Format::uri-reference
             JSON-Suite::Optional::Format::uri-template
             JSON-Suite::Optional::Format::uri
+            JSON-Suite::Optional::Format::uuid
 
```

Comment:
The test should not be added here: these are the tests expected to fail. `WILL_FAIL ON`.

Reply:
I wasn't quite sure where to add the tests. Can you provide some instructions?

Reply:
Nowhere, it is found by a GLOB-call from CMake.